ui/ducks/institutional/institutional.js

```diff
@@ -0,0 +1,114 @@
+import { createSelector } from 'reselect';
+import { createSlice } from '@reduxjs/toolkit';
+import { captureException } from '@sentry/browser';
+import { mmiActionsFactory } from '../../store/institutional/institution-background';
+
+const createComplianceActionType = (action) =>
+  `metamask/institutional-features/compliance/${action}`;
+
+const SET_HISTORICAL_REPORTS = createComplianceActionType(
+  'SET_HISTORICAL_REPORTS',
+);
+
+const name = 'institutionalFeatures';
+
+const initialState = {
+  historicalReports: {},
+  complianceProjectId: '',
+  complianceClientId: '',
+  reportsInProgress: {},
+};
+
+const slice = createSlice({
+  name: 'reports',
```

Comment:
reports is a strange name for the 'institutional' slice. note that actions defined in reducers are automatically given types, so if you name your slice 'mmi' the action type will be 'mmi/setHistoricalReports' ... the definition of SET_HISTORICAL_REPORTS above is redundant and probably doesn't work the way you expect.

Reply:
Indeed, I'll use the value already defined in the const name. (institutionalFeatures)